tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_volume_scenarios.py

```diff
@@ -0,0 +1,199 @@
+import random
+import time
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+"""
+ceph fs volume rename <vol_name> <new_vol_name> [--yes-i-really-mean-it]
+# Delete Scenarios
+1. Create a volume
+2. Try to delete the volume wihh mon_allow_pool_delete to false
+3. Check volume delete fails and validate the error message
+4. Set mon_allow_pool_delete to true
+5. Delete the volume and check if the volume is deleted
+# Rename scenarios
+6. Try to rename the volume
+7. Try to rename the volume with additional pools
+8. Try to rename the volume with subvolumegroup and subvolume
+9. Rename the volume when the volume is down -> fail fs
+10. Rename the volume with and without refuse_client_session flags
+
+"""
+
+
+def run(ceph_cluster, **kw):
+    try:
+        tc = "CEPH-83603354"
+        log.info(f"Running CephFS tests for - {tc}")
+        # Initialize the utility class for CephFS
+        fs_util = FsUtils(ceph_cluster)
+        # Get the client nodes
+        clients = ceph_cluster.get_ceph_objects("client")
+        config = kw.get("config")
+        # Authenticate the clients
+        fs_util.auth_list(clients)
+        build = config.get("build", config.get("rhbuild"))
+        # Prepare the clients
+        fs_util.prepare_clients(clients, build)
+        client1 = clients[0]
+        ran_string = "".join(random.choices("abcdefghijklmnopqrstuvwxyz", k=5))
+        start_volume = f"cephfs1_{ran_string}"
+        client1.exec_command(sudo=True, cmd=f"ceph fs volume create {start_volume}")
```

Comment:
Can we use common function https://github.com/red-hat-storage/cephci/blob/7a240fdbe5ec87227f162c97efcd925de8740d56/tests/cephfs/cephfs_utilsV1.py#L1413

This way any command level changes can be made at one point